rust/datafusion/src/physical_plan/functions.rs

```diff
@@ -1033,6 +1163,54 @@ mod tests {
 
     #[test]
     fn test_functions() -> Result<()> {
+        test_function!(
+            Ascii,
+            &[lit(ScalarValue::Utf8(Some("x".to_string())))],
+            Ok(Some(120)),
+            i32,
+            Int32,
+            Int32Array
+        );
+        test_function!(
+            Ascii,
+            &[lit(ScalarValue::Utf8(Some("ésoj".to_string())))],
+            Ok(Some(233)),
+            i32,
+            Int32,
+            Int32Array
+        );
+        test_function!(
+            Ascii,
+            &[lit(ScalarValue::Utf8(Some("💯".to_string())))],
+            Ok(Some(128175)),
```

Comment:
It is strange to me that a function called `ascii` can return something larger than `127`. However, it seems that quirkiness / awesomeness came from `postgres` :)

```
alamb=# select ascii('💯');
 ascii  
--------
 128175
(1 row)

```
👍